torch/_tensor.py

```diff
@@ -81,6 +81,7 @@ def _rebuild_from_type_v2(func, new_type, args, state):
 class Tensor(torch._C.TensorBase):
     _dynamo_static_input_type: str  # Set in `mark_static_address`
     _dynamo_dynamic_indices: Set[int]  # Set in `mark_dynamic`
+    _is_param: bool  # Set on torch.nn.Parameter instances
```

Comment:
What is the reason for adding this?

Reply:
It's set here: 

https://github.com/pytorch/pytorch/blob/main/torch/_inductor/freezing.py#L154

and here:

https://github.com/pytorch/pytorch/blob/main/torch/_inductor/freezing.py#L169